stratum/hal/config/BUILD

```diff
@@ -11,7 +11,7 @@ load(
 load(":platform_config_test.bzl", "platform_config_test")
 load("@rules_pkg//:pkg.bzl", "pkg_tar", "pkg_deb")
 
-package(default_visibility = ["//visibility:public"])
+package(default_visibility = STRATUM_INTERNAL)
```

Comment:
@Yi-Tseng Do you remember why you changed this to public?

Reply:
I think I was trying to use create some tools with Bazel to use these configs. But I didn't add those tests to this repo in the end and forgot to revert this change.

Reply:
I think I was trying to use create some tools with Bazel to use these configs. But I didn't add those tests to this repo in the end and forgot to revert this change.

Reply:
Ok, with this change new tools can live under `/stratum` and we can leave this private.